package.json

```diff
@@ -2,9 +2,10 @@
   "name": "electron-pdf",
   "version": "0.12.0",
   "description": "A command line tool to generate PDF from URL, HTML or Markdown files",
-  "main": "index.js",
+  "main": "lib/index.js",
   "scripts": {
-    "test": "standard"
+    "lint": "standard",
+    "test": "standard && node_modules/.bin/electron node_modules/.bin/jasmine"
```

Comment:
Two cents:

1. `npm run` script will auto attach `node_modules/.bin` into the $PATH so you don't need it in `scripts` field of `package.json`;
2.  Why do you use electron to run jasmine? Do you mean `standard && jasmine`?